### PR TITLE
Integer input directive

### PR DESCRIPTION
**Pull request by jrchudy**
This PR addresses issue #806.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -134,33 +134,34 @@ JS_DEPS=$(JS)/vendor/jquery-latest.min.js \
...
-	$(JS)/facetsModel.js \
-	$(JS)/facetsService.js \
 	$(JS)/controller/ermrestDetailController.js \
 	$(JS)/controller/ermrestFilterController.js \
 	$(JS)/controller/ermrestInitController.js \
 	$(JS)/controller/ermrestLoginController.js \
 	$(JS)/controller/ermrestResultsController.js \
 	$(JS)/controller/ermrestSideBarController.js \
 	$(JS)/controller/ermrestTourController.js \
+	$(JS)/ermrest.js \
```
Comment (RFSH) [inline Makefile]: It seems like you have changed the order of the resources. I'm just curious, was that necessary? or you just wanted to organize them?

File: common/inputs.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                        scope.max = 9223372036854775807;
+                        break;
+                    default:
+                        break;
+                }
+
+                if (!ngModel) return;
+
+                scope.onChange = function(){
+                    ngModel.$setViewValue(scope.value);
```
Comment (RFSH) [inline common/inputs.js]: I can see that at some point we might want to call a function on input change. We can pass a callback to the directive which here will be called.

File: Makefile (hunk 1 above)
Comment (jrchudy) [inline Makefile]: I was organizing them better and didn't realize I changed the `JS_RESOURCE` at first. The order does matter for some of the resources though (jquery needs to be defined first before angular for instance).

File: common/inputs.js (hunk 2 above)
Comment (jrchudy) [inline common/inputs.js]: I agree. I will make this a callback.

**Comment by jrchudy** [comment]
This PR is no longer relevant.

**Pull request closed without merging**